frontend/css/styles.css.map

```diff
@@ -1,6 +1,6 @@
,SAAS,EAAE,gBAAgB;MAC3B,QAAQ,EAAE,QAAQ;MAClB,OAAO,EAAE,EAAE;MACX,OAAO,EAAE,GAAG;MKsJJ,UAAU,EAAE,uBAAuB;IAOnC,oHAAgB;MACZ,SAAS,EAAE,aAAa;EAKpC,+BAAc;IAEV,MAAM,EAAE,CAAC;IACT,OAAO,EC3FL,QAAmD;ID4FrD,cAAc,EC5FZ,OAAmD;ID6FrD,OAAO,EAAE,GAAG;IACZ,OAAO,EAAE,EAAE;IAEX,SAAS,EAAE,gBAAgB;IAC3B,UAAU,EAAE,aAAa;IAEzB,kFACa;MACT,SAAS,EAAE,aAAa;IAG5B,0CAAW;ML9Kf,SAAS,EMsEH,QAAmD;MNrEzD,WAAW,EXbH,uBAAuB;MWc/B,WAAW,EAAE,MAAM;MACnB,KAAK,EVgcS,OAAgB;;Ae9QlC;0BAC2B;EAEvB,KAAK,EjBxLC,OAAO;EiB2Lb;4BAAgB;IACZ,gBAAgB,EfuQN,OAAgB;IetQ1B;qCAAS;MACL,mBAAmB,EfqQb,OAAgB;EejQ9B;4BAAkB;IACd,gBAAgB,EfstCS,OAAU;IertCnC;qCAAS;MACL,mBAAmB,EfotCE,OAAU;EehtCvC;4BAAiB;IACb,gBAAgB,EjBrKP,OAAM;IiBsKf;qCAAS;MACL,mBAAmB,EjBvKd,OAAM;EiB2KnB;4BAAmB;IACf,gBAAgB,EjB3KN,OAAQ;IiB4KlB;qCAAS;MACL,mBAAmB,EjB7Kb,OAAQ;;AiB0L1B,cAAe;EAGX,gBAAgB,EfsrCa,OAAU;EerrCvC,QAAQ,EAAE,QAAQ;EAElB,qBAAS;ILvPT,KAAK,EAAE,CAAC;IACR,MAAM,EAAE,CAAC;IAET,WAAW,EAAE,0BAAyC;IACtD,YAAY,EAAE,0BAAyC;IACvD,aAAa,EAAE,sBAAqC;IACpD,GAAG,EAAE,UAA8B;IAEnC,IAAI,EAAE,GAAG;IACT,SAAS,EAAE,gBAAgB;IAC3B,QAAQ,EAAE,QAAQ;IAClB,OAAO,EAAE,EAAE;IACX,OAAO,EAAE,GAAG;;AKgPhB,eAAgB;EACZ,OAAO,EAAE,IAAI;EACb,cAAc,EAAE,GAAG;EACnB,SAAS,EAAE,IAAI;EACf,SAAS,EAAE,IAAI;;AAGnB,UAAW;EACP,KAAK,EjBnMU,OAAM;EiBoMrB,SAAS,EC9KH,QAAmD;ED+KzD,WAAW,EhBjQH,uBAAuB;EgBkQ/B,KAAK,EjBtMU,OAAM;EiBuMrB,IAAI,EAAE,IAAI;EACV,UAAU,EAAE,IAAI;EAChB,OAAO,ECnLD,QAAmD;;ADwL7D,UAAW;EL9PP,SAAS,EMsEH,QAAmD;ENrEzD,WAAW,EXbH,uBAAuB;EWc/B,WAAW,EAAE,MAAM;EACnB,KAAK,EVgcS,OAAgB;;AejMlC,aAAc;EACV,SAAS,EC7LH,QAAmD;ED8LzD,WAAW,EhBhRH,uBAAuB;EgBiR/B,WAAW,EAAE,OAAO;EACpB,KAAK,EjBtNU,OAAM;EiBuNrB,IAAI,EAAE,IAAI;EACV,OAAO,EClMD,QAAmD;;ADuM7D,YAAa;EACT,IAAI,EAAE,CAAC;EACP,MAAM,EAAE,GAAG;EACX,gBAAgB,EfsoCgB,OAAwB;EeroCxD,OAAO,EAAE,IAAI;EACb,cAAc,EAAE,MAAM;EACtB,MAAM,EAAE,IAAI;;AAIhB,kBAAmB;EACf,UAAU,EA1DQ,QAAY;EA2D9B,WAAW,EA3DO,QAAY;EA4D9B,gBAAgB,EfyKF,OAAgB;EexK9B,UAAU,EAAE,MAAM;EAClB,MAAM,EAAE,CAAC;EACT,IAAI,EAAE,CAAC;EACP,WAAW,EAAE,QAAQ;EACrB,KAAK,EfunC2B,OAAwB;EetnCxD,WAAW,EAAE,IAAI;;AAarB,eAAgB;EAEZ,gBAAgB,EjB9QC,OAAK;EiB+QtB,QAAQ,EAAE,QAAQ;EAElB,sBAAS;ILnUT,KAAK,EAAE,CAAC;IACR,MAAM,EAAE,CAAC;IAET,WAAW,EAAE,0BAAyC;IACtD,YAAY,EAAE,0BAAyC;IACvD,aAAa,EAAE,sBAAqC;IACpD,GAAG,EAAE,UAA8B;IAEnC,IAAI,EAAE,GAAG;IACT,SAAS,EAAE,gBAAgB;IAC3B,QAAQ,EAAE,QAAQ;IAClB,OAAO,EAAE,EAAE;IACX,OAAO,EAAE,GAAG;;AKiUhB,WAAY;EAGR,gBAAgB,EfoIF,OAAgB;EenI9B,QAAQ,EAAE,QAAQ;EAElB,kBAAS;ILnVT,KAAK,EAAE,CAAC;IACR,MAAM,EAAE,CAAC;IAET,WAAW,EAAE,0BAAyC;IACtD,YAAY,EAAE,0BAAyC;IACvD,aAAa,EAAE,sBAAqC;IACpD,GAAG,EAAE,UAA8B;IAEnC,IAAI,EAAE,GAAG;IACT,SAAS,EAAE,gBAAgB;IAC3B,QAAQ,EAAE,QAAQ;IAClB,OAAO,EAAE,EAAE;IACX,OAAO,EAAE,GAAG;;AOjBhB;;uCAEuC;AAEvC,WAAY;EAER,WAAW,EDsFL,SAAmD;ECrFzD,cAAc,EDqFR,MAAmD;ECpFzD,gBAAgB,EjBo6CgB,OAAwB;;AkB56C5D;;uCAEuC;AAEvC,cAAe;EAEX,WAAW,EFsFL,SAAmD;EErFzD,cAAc,EFqFR,MAAmD;EEpFzD,gBAAgB,ElBo6CgB,OAAwB;;AmBr4C5D,IAAK;EACJ,WAAW,EpB9BA,uBAAuB;EoB+BlC,WAAW,EpBhCE,GAAG;;AoBoCjB,CAAE;EACE,eAAe,EAAE,IAAI;EACrB,WAAW,EpBtCD,GAAG;;AoBwCjB,aAAc;EACb,SAAS,EClDE,IAAI;EDmDf,MAAM,EAAE,MAAM;;AAGf,IAAK;EACJ,SAAS,EAAE,IAAI",
```

Comment:
It is auto generated by sass, the explanation on what it is for is here: https://developer.chrome.com/devtools/docs/css-preprocessors basically to help see where the code originally came form in dev tools.